docs/platform/integration/commercetools/overview.md

```diff
@@ -1,70 +1,29 @@
 ---
 id: overview
-title: CommerceTools Integration
+title: commercetools Integration
 sidebar_label: Overview
 ---
 
 import CodePackage from '@site/src/components/CodePackage';
 
 <CodePackage name="@deity/falcon-commercetools-module" />
 
-## Installation
+### commercetools introduction
+- commercetools, a headless e-commerce platform
+- ctpSession - Hold http session data
+- ctpClient - Connect to SDK in name of user or integration, using ctpSession for authentication
```

Comment:
```suggestion
- Commercetools SDK integration ([ctpClient](LINK TO SDK DOCS HERE))
```

Reply:
TBH i'm not sure these bullet points make too much sense. I think a short paragraph would make more sense